scripts/globals/effects/carol.lua

```diff
@@ -7,14 +7,14 @@
 require("scripts/globals/status");
 require("scripts/globals/magic");
 
-local resistMod = {dsp.mod.FIRERES, dsp.mod.EARTHRES, dsp.mod.WATERRES, dsp.mod.WINDRES, dsp.mod.ICERES, dsp.mod.THUNDERRES, dsp.mod.LIGHTRES, dsp.mod.DARKRES};
+local dsp.magic.resistMod = {dsp.mod.FIRERES, dsp.mod.EARTHRES, dsp.mod.WATERRES, dsp.mod.WINDRES, dsp.mod.ICERES, dsp.mod.THUNDERRES, dsp.mod.LIGHTRES, dsp.mod.DARKRES};
```

Comment:
for local variables we don't need to namespace them under dsp, only globals (luacheck doesn't seem to agree with this bring local variable either)